internal/connect/command_plugin_install.go

```diff
@@ -51,7 +51,7 @@ func newInstallCommand(prerunner pcmd.PreRunner) *cobra.Command {
 				Code: "confluent connect plugin install confluentinc/kafka-connect-datagen:latest --plugin-directory $CONFLUENT_HOME/plugins --worker-configurations $CONFLUENT_HOME/etc/kafka/connect-distributed.properties",
 			},
 		),
-		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonCloudLogin},
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNotCloudLogin},
```

Comment:
Alternate language suggestion since "Non" and "Not" differ by one letter and might be confusing:
```suggestion
		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLogout},
```